README.md

```diff
@@ -80,7 +80,18 @@ $ tail -f $XDG_STATE_HOME/filecoin-station/logs/modules/*.log
 ...
 ```
 
-## Deployment
+## Docker
+
+Deploy Station with [Docker](https://www.docker.com/). Please replace
+`FIL_WALLET_ADDRESS`.
+
+```bash
+$ docker run \
+	-e FIL_WALLET_ADDRESS=f1abjxfbp274xpdqcpuaykwkfb43omjotacm2p3za \
+	-d ghcr.io/filecoin-station/core
+```
+
```

Comment:
Could you please add an example of obtaining metrics and list logs?

I think it would be something along the following lines:


```
docker exec ghcr.io/filecoin-station/core ./bin/station.js metrics
```

Docker docs: https://docs.docker.com/engine/reference/commandline/exec/

Reply:
great idea, done 🙏